[On Hold] Add Heliarch and Resistance intros (#4800)
endless-sky/endless-sky: data/coalition/coalition outfits.txt

	"turning heat" 8.2
	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships, but gains a considerable amount of flexibility."

outfit "Refuelling Module"
	category "Systems"
	cost 822000
	thumbnail "outfit/refuelling module"
	"mass" 9
	"outfit space" -9
	"fuel capacity" -100
	"fuel generation" .05
	"energy consumption" .025

> 	description "To avoid the risk of being inspected by the Heliarch as they refuel on planets, Resistance ships employ this outfit when carrying supplies or stolen outfits."

Reviewer: ```suggestion
	description "To avoid the risk of being inspected by the Heliarch as they refuel on planets, the Resistance have created an outfit capable of refuelling a ship in the middle of flight for ships carrying contraband."
```

Author: didn't do exactly those changes

> 	description "	The outfit's mechanisms require a set amount of fuel to be put aside for its inner workings, so a ship can only install so many of these before becoming incapable of jumping to another system."

Reviewer: ```suggestion
	description "	The outfit's mechanisms require a set amount of fuel to be put aside for its inner workings, so a ship can only install so many of these before it doesn't have enough fuel to jump to another system."
```

outfit "Decoy Plating"
	category "Systems"
	cost 93000
	thumbnail "outfit/decoy plating"
	"mass" 7
	"outfit space" -7
	"scan interference" 1.2
	"radar jamming" 4

> 	description "This plating takes advantage of a ship's scan logs, reproducing the data of other ships close by and effectively deflecting attempts to scan a ship's true cargo and equipment."

Reviewer: ```suggestion
	description "This special plating is able to take advantage of a ship's scan logs. By mimicking the scan data of other ships close by, this plating is able to effectively hide a ship's true cargo and equipment from scanning."
```

Author: didn't want "is able to" to be repeating so didn't do that on the second instance

> 	description "	To protect themselves against the destructive Heliarch torpedoes, the Resistance made sure to also incorporate a radar jamming system in it, if only to provide the ship with a final line of defense."

Reviewer: ```suggestion
	description "	Additionally, as a last-ditch effort to protect against the destructive Heliarch Finishers, the Resistance has also incorporated radar jamming into the plating."
```

	category "Special"
	thumbnail "outfit/coalition license"
	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships."

outfit "Heliarch License"
	category "Special"
	thumbnail "outfit/heliarch license"

> 	description "Assisted the Heliarch in several instances, and as such was awarded the rank of Heliarch. This circlet was originally designed for a Saryd, but was modified to better fit a human head."

Author: their description is `And the third species are centaurs. Not just something vaguely resembling centaurs; they look like they could have stepped right off the page of a story book from ancient Earth.` , while true that as such the upper body should be exactly that of a human, "right off the page of a story book" rather than "right off of Greek mythology" leaves space for them being centaurs as are described in fiction which may not depict them as being literally the Greek centaurs. I personally prefer suggesting that there *are* physical differences between the upper body of a Saryd and that of a human, even if it's just size or some details that would affect how a circlet is made, because a perfectly human upper body seems a bit immersion-breaking to me
